pkg/core/blockchain.go

```diff
@@ -150,7 +150,7 @@ type Blockchain struct {
 
 	// postBlock is a set of callback methods which should be run under the Blockchain lock after new block is persisted.
 	// Block's transactions are passed via mempool.
-	postBlock []func(blockchainer.Blockchainer, *mempool.Pool, *block.Block)
+	postBlock []func(func(*transaction.Transaction, *mempool.Pool, bool) bool, *mempool.Pool, *block.Block)
```

Comment:
Maybe create a delegate for it? Thes long signature is used in multiple places.

Reply:
The problem is where to put this definition, `core` should have it, but it can't, because `network` can't import it. Moving to `blockchainer` isn't nice either. I think we better have it now as is.